.github/workflows/zxc-xts-tests.yaml

```
@@ -151,138 +163,51 @@ jobs:
-    name: XTS HAPI Tests (Token)
-    if: ${{ inputs.enable-extended-test-suite == 'true' }}
-    uses: ./.github/workflows/node-zxc-compile-application-code.yaml
-    with:
-      custom-job-label: "${{ inputs.custom-job-label }} HAPI Token"
-      enable-hapi-tests-token: true
-      enable-network-log-capture: true
-      ref: ${{ inputs.commit_sha }}
-    secrets:
-      access-token: ${{ secrets.access-token }}
-      gradle-cache-username: ${{ secrets.gradle-cache-username }}
-      gradle-cache-password: ${{ secrets.gradle-cache-password }}
-
-  xts-hapi-smart-contract-tests:
-    name: XTS HAPI Tests (Smart Contract)
-    if: ${{ inputs.enable-extended-test-suite == 'true' }}
-    uses: ./.github/workflows/node-zxc-compile-application-code.yaml
-    with:
-      custom-job-label: "${{ inputs.custom-job-label }} HAPI Smart Contract"
-      enable-hapi-tests-smart-contract: true
-      enable-network-log-capture: true
-      ref: ${{ inputs.commit_sha }}
-    secrets:
-      access-token: ${{ secrets.access-token }}
-      gradle-cache-username: ${{ secrets.gradle-cache-username }}
-      gradle-cache-password: ${{ secrets.gradle-cache-password }}
-
-  xts-hapi-block-node-comms-tests:
-    name: XTS HAPI Tests (Block Node Communication)
-    if: ${{ inputs.enable-extended-test-suite == 'true' }}
-    uses: ./.github/workflows/node-zxc-compile-application-code.yaml
+  xts-hapi-tests:
+    name: XTS HAPI Tests
+    needs:
+      - build
+    if: ${{ needs.build.result == 'success' && inputs.enable-extended-test-suite == 'true' }}
+    uses: ./.github/workflows/zxc-execute-hapi-tests.yaml
     with:
-      custom-job-label: "${{ inputs.custom-job-label }} HAPI Block Node Comms"
-      enable-hapi-tests-bn-communication: true
-      enable-network-log-capture: true
+      custom-job-label: "${{ inputs.custom-job-label }} HAPI Tests"
       ref: ${{ inputs.commit_sha }}
+      enable-hapi-tests-time-consuming: "true"
+      enable-hapi-tests-misc: "true"
+      enable-hapi-tests-misc-records: "true"
+      enable-hapi-tests-crypto: "true"
+      enable-hapi-tests-simplefees: "true"
+      enable-hapi-tests-token: "true"
+      enable-hapi-tests-smart-contract: "true"
+      enable-hapi-tests-bn-communication: "true"
+      enable-network-log-capture: "true"
     secrets:
       access-token: ${{ secrets.access-token }}
       gradle-cache-username: ${{ secrets.gradle-cache-username }}
       gradle-cache-password: ${{ secrets.gradle-cache-password }}
 
   xts-otter-tests:
     name: XTS Otter Tests
-    if: ${{ inputs.enable-extended-test-suite == 'true' }}
+    needs:
+      - build
+    if: ${{ needs.build.result == 'success' && inputs.enable-extended-test-suite == 'true' }}
     uses: ./.github/workflows/zxc-execute-otter-tests.yaml
     with:
       custom-job-label: "${{ inputs.custom-job-label }} Otter Tests"
       enable-full-otter-tests: true
-      enable-network-log-capture: true
+      enable-network-log-capture: "true"
```

Is there a reason you have quotes here?

```suggestion
      enable-network-log-capture: true
```

---

Parameter changed to a string.